### PR TITLE
- [gst-plugins-bad] add 1.19.1

### PR DESCRIPTION
**Pull request by SSE4**
closes: #8748 

Specify library name and version:  **gst-plugins-bad/1.19.1**

This is also a good place to share with all of us **why you are submitting this PR** (specially if it is a new addition to ConanCenter): is it a dependency of other libraries you want to package? Are you the author of the library? Thanks!

---

- [ ] I've read the [guidelines](https://github.com/conan-io/conan-center-index/blob/master/docs/how_to_add_packages.md) for contributing.
- [ ] I've followed the [PEP8](https://www.python.org/dev/peps/pep-0008/) style guides for Python code in the recipes.
- [ ] I've used the [latest](https://github.com/conan-io/conan/releases/latest) Conan client version.
- [ ] I've tried at least one configuration locally with the
      [conan-center hook](https://github.com/conan-io/hooks.git) activated.

### CODE REVIEW

File: recipes/gst-plugins-bad/all/conanfile.py [hunk 1]
```diff
@@ -0,0 +1,232 @@
...
+        del self.settings.compiler.cppstd
+        self.options['gstreamer'].shared = self.options.shared
+        self.options['gst-plugins-base'].shared = self.options.shared
+
+    def config_options(self):
+        if self.settings.os == 'Windows':
+            del self.options.fPIC
+
+    def requirements(self):
+        self.requires("glib/2.70.0")
```
Comment (SSE4) [inline recipes/gst-plugins-bad/all/conanfile.py:66]:
```suggestion
        self.requires("glib/2.70.1")
```

**Review by uilianries — APPROVED** [review]
LGTM

**Review by jgsogo — APPROVED** [review]
Let's merge, even if these are _bad_ plugins 😄

**Review by prince-chrismc — APPROVED** [review]

**Comment by prince-chrismc** [comment]
> Please review this pull request, we would like to use this recipe in our build.

Sorry the review time have gotten really slow 😭

**Pull request merged**